From: 2ed-code <2ed-code@users.noreply.github.com>
Date: Thu, 17 Sep 2026 00:00:00 +0000
Subject: 207200 branding

Set the Chromium product name and short product name to 207200.

---
 chrome/app/chromium_strings.grd | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/chrome/app/chromium_strings.grd b/chrome/app/chromium_strings.grd
--- a/chrome/app/chromium_strings.grd
+++ b/chrome/app/chromium_strings.grd
@@ -297,12 +297,12 @@
   </message>
   </then>
   <else>
   <message name="IDS_PRODUCT_NAME" desc="The Chrome application name" translateable="false">
-        Chromium
+        207200
   </message>
   <message name="IDS_SHORT_PRODUCT_NAME" desc="The Chrome application short name." translateable="false">
-        Chromium
+        207200
   </message>
   </else>
   </if>
--
